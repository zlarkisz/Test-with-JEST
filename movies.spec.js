import movies from "./movies";

describe("Favorite Movies", () => {
  let myMovies = [];

  beforeEach(() => {
    myMovies = [
      {
        title: "The Matrix",
        rate: null,
      },
    ];
  });

  test.only("can add a movie", () => {
    movies.add(myMovies, "Kung Fu Panda");
    expect(myMovies).toMatchSnapshot();
  });

  test.skip("rate a movie", () => {
    movies.rate(myMovies[0], 5);
    expect(myMovies).toMatchSnapshot();
  });
});
