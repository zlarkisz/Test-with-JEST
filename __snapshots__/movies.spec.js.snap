// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorite Movies can add a movie 1`] = `
[
  {
    "rate": null,
    "title": "The Matrix",
  },
  {
    "rate": null,
    "title": "Kung Fu Panda",
  },
]
`;

exports[`Favorite Movies rate a movie 1`] = `
[
  {
    "rate": 5,
    "title": "The Matrix",
  },
]
`;
